web/src/ResourceStatusSummary.tsx

```diff
@@ -283,31 +283,38 @@ type ResourceStatusSummaryProps = {
 }
 
 export function ResourceStatusSummary(props: ResourceStatusSummaryProps) {
-  // Count the statuses.
+  // Count and calculate the combined statuses.
   let resources = props.view.uiResources || []
+  const allStatuses: ResourceStatus[] = []
 
   let testResources = new Array<UIResource>()
+  const testStatuses: ResourceStatus[] = []
   let otherResources = new Array<UIResource>()
+  const otherStatuses: ResourceStatus[] = []
```

Comment:
it looks like `testResources` and `otherResources` are now unused and can be removed?

Reply:
yes! i will fix.